app/controllers/shf_documents_controller.rb

```diff
@@ -2,11 +2,12 @@ class ShfDocumentsController < ApplicationController
 
   before_action :set_shf_document, only:  [ :show, :edit, :update, :destroy ]
   before_action :authorize_shf_doc, only: [ :show, :edit, :update, :destroy ]
-  before_action :authorize_shf_doc_class, only: [:index, :new, :create, :historika_meeting_minutes ]
```

Comment:
`:historika_meeting_minutes` which I assume is "protokoll" is missing, is it not needed anymore?